packages/health-and-info/src/types/HealthCheckOptions.ts

```diff
@@ -0,0 +1,107 @@
+import { HttpOptions, HttpsOptions } from 'agentkeepalive'
```

Comment:
**question:**  I noticed in a few places you could `import type`, is there a reason for not doing it? (not sure whether typescript is better now at detecting where "only types" are used)

Reply:
When compiling down, this doesn't lead to any extraneous imports of function/implementation, so it's aware it's purely just for typing. Typescript is pretty good these days at detecting specifically when an import is used just for a type, so I don't think this one is necessary 😃